Alignment/CommonAlignmentProducer/plugins/PCLTrackerAlProducer.cc

```diff
@@ -507,7 +507,7 @@ ::createGeometries(const edm::EventSetup& setup)
     setup.get<PTrackerParametersRcd>().get( ptp );
 
     theTrackerGeometry = boost::shared_ptr<TrackerGeometry>(
-        trackerBuilder.build(&(*geometricDet), *ptp )
+        trackerBuilder.build(&(*geometricDet), *ptp, tTopo )
```

Comment:
@ebrondol - IMHO, it is better to clean up BIG_PIX_PER_ROC_X/Y
I do not see them used in RectangularPixelTopology http://cmslxr.fnal.gov/source/Geometry/TrackerGeometryBuilder/interface/RectangularPixelTopology.h?v=CMSSW_8_0_X_2016-01-19-2300